internal/schema2/property_type.go

```diff
@@ -18,6 +18,7 @@ const (
 	PTProcessList                 PropertyType = "ProcessList"
 	PTTerminateOnLastHandleClosed PropertyType = "TerminateOnLastHandleClosed"
 	PTSharedMemoryRegion          PropertyType = "SharedMemoryRegion"
+	PTContainerCredentialGuard    PropertyType = "ContainerCredentialGuard" // This field is not generated by swagger. This was added manually.
```

Comment:
Can we generate this again from an updated source, rather than adding the new definition manually?

Reply:
I'll ping you offline about this

Reply:
@kevpar Is this ok to leave for now as we find a way around the discussed issue?

Reply:
Yeah, we followed up offline. It's fine to manually edit the generated files to resolve issues like this for now.